docs/modules/ROOT/pages/usage.adoc

```diff
@@ -0,0 +1,47 @@
+= Usage
+
+The operator injects secret data into xref:volume.adoc[] mounts that declare a CSI volume with `driver: secrets.stackable.tech`.
+  
+A minimal secret-consuming `Pod` looks like this:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: example-secret-consumer
+spec:
+  volumes:
+  - name: secret
+    csi:
+      driver: secrets.stackable.tech
+      volumeAttributes:
+        secrets.stackable.tech/class: tls
+        secrets.stackable.tech/scope: node,pod,service=secret-consumer-nginx
+  containers:
+  - name: ubuntu
+    image: ubuntu
+    stdin: true
+    tty: true
+    volumeMounts:
+    - name: tls
```

Comment:
Does this need to be `secret` instead of `tls`?

Reply:
Good point, this should be changed to `tls` in the volume definition.